blobfuse-nightly.yml

```diff
@@ -874,6 +874,8 @@ jobs:
 
         # Prebuild cleanup
         - script: |
+            sudo fusermount -u $(MOUNT_DIR)
+            sudo kill -9 `pidof blobfuse`
```

Comment:
It will give an error but step will not fail. This is just a safe check, in case we cancel a running job then blobfuse remain mounted on self hosted agents and next time we try to run the pipeline it fails. So first we cleanup the last run and then proceed.